docs/cli/argo_archive_list.md

```diff
@@ -6,6 +6,22 @@ list workflows in the archive
 argo archive list [flags]
 ```
 
+### Examples
+
+```
+# List all Archived Workflows:
+  argo archive list
+
+# List Archived Workflows in YAML format:
+  argo archive list --output yaml
+
+# List Archived Workflows with a specific chunk size:
+  argo archive list --chunk-size <chunk_size>
+
+# List Archived Workflows with the specified label (used to filter archived workflows):
```

Comment:
this doesn't match the modified description (you changed it to label_s_, plural)